.editorconfig

```diff
@@ -0,0 +1,20 @@
+root = true
```

Comment:
Yeah, we're adding it to all projects (see standards posts) and I figured while I was mucking about might as well. Just makes tab spacing consistent in Github displays.